src/pybind/mgr/dashboard/module.py

```diff
@@ -125,7 +126,8 @@ def _configure(self):
 
         :returns our URI
         """
-        server_addr = self.get_localized_module_option('server_addr', '::')
+        server_addr = self.get_localized_module_option('server_addr',
+                mgr_util.get_default_addr())
```

Comment:
The indention is incorrect.
```
py3-lint run-test: commands[2] | pylint --rcfile=.pylintrc --jobs=5 . module.py tools.py controllers tests services exceptions.py
************* Module dashboard.module
module.py:130:0: C0330: Wrong continued indentation (add 39 spaces).
                mgr_util.get_default_addr())
                ^                                      | (bad-continuation)

------------------------------------
Your code has been rated at 10.00/10
```
Please run ``run-tox.sh`` before committing a PR.

Reply:
ok